src/js/utils/ServiceUtil.js

```diff
@@ -84,10 +84,12 @@ const ServiceUtil = {
       }
 
       if (labels != null) {
-        definition.labels = labels.labels.reduce(function (memo, item) {
-          memo[item.key] = item.value;
-          return memo;
-        }, {});
+        if (labels.labels != null) {
```

Comment:
You are right, I thought about keeping it this way in case something gets added later to that section, which is probably not the case. But since you mentioned it I changed it to be only one condition.